README.md

```diff
@@ -127,3 +131,12 @@ Options:
 
 
 ```
+Configuration options can also be stored as json in `truffle-security.json` at the truffle project root. i.e. : 
+```
+{
+    "style": "table",
+    "mode": "quick",
+    "min-severity": "warning",
+    "swc-blacklist": "103,111"
```

Comment:
Thanks @nbanmp for taking a look here. I had tested it to work with strings https://github.com/ConsenSys/truffle-security/pull/139/files#diff-efa555d3a5a4caad62d0d92287e43158R849, but am happy to update to an array as input for the blacklist if you feel that would be clearer?

Reply:
Hmm... When I tested it with commas in the `truffle-security.json` file, it still does not seem to successfully filter. Could you check that out?

Reply:
@nbanmp  Ah darn, sorry about that. I haven't been able to reproduce that, yet I wonder:
was only the json file version broken for the blacklist or was the cli usage not working as well?

The only way I was able to break the comma separated functionality was by passing up invalid json. Aside from that I'm not sure how else to isolate the difference other than maybe taking advantage of of the integration-tests directory to isolate for environment/solc version, etc... 

Will continue to brainstorm if it might be something else :thinking:

Reply:
Tested with `"swc-blacklist": "103,111"` in truffle-security.json and filtering didn't work for me too, but `"swc-blacklist": [103,111]` or `"swc-blacklist": 103` worked for me. Fixing `"swc-blacklist": "103,111"` to `"swc-blacklist": [103,111]  is enough. And for me personally it is much nicer to have an array rather than a string.

Reply:
@nbanmp does the readme update here work for you? any other changes you'd like to see before marking this as done? thanks!